### PR TITLE
Add Simple Control Flow Support

### PR DESCRIPTION
**Pull request by jiangjiajun**
Add `while` support(Only part of situations), also modify `slice`
Thanks for contributing to TVM!   Please refer to guideline https://tvm.apache.org/docs/contribute/ for useful information and tips. After the pull request is submitted, please request code reviews from [Reviewers](https://github.com/apache/incubator-tvm/blob/master/CONTRIBUTORS.md#reviewers) by @ them in the pull request thread.

### CODE REVIEW

File: python/tvm/relay/frontend/paddlepaddle.py [hunk 1]
```diff
@@ -1349,60 +1424,72 @@ def convert_slice(g, op, block):
 
     data = g.get_node(op.input("Input")[0])
     dims = len(infer_shape(data))
-    dtype = "int64"
 
     axes = op.attr("axes")
-    axes = _op.const(axes)
+    indices = _expr.const(axes).astype("int64")
```
Comment (heliqi) [inline python/tvm/relay/frontend/paddlepaddle.py:1429]: const是能传dtype， 就不用多个cast的op了

File: python/tvm/relay/frontend/paddlepaddle.py [hunk 2]
```diff
@@ -1349,60 +1424,72 @@ def convert_slice(g, op, block):
...
+        starts = _infer_value(starts, g.get_params())
     elif op.input("StartsTensorList"):
         starts = []
         for start_index in op.input("StartsTensorList"):
-            start_index = g.get_node(start_index)
-            if not isinstance(start_index, _expr.Expr):
-                start_index = _expr.const(start_index, dtype=dtype)
-            else:
-                start_index = start_index.astype(dtype)
+            start_index = g.get_node(start_index).astype("int64")
```
Comment (heliqi) [inline python/tvm/relay/frontend/paddlepaddle.py:1441]: 同上

File: python/tvm/relay/frontend/paddlepaddle.py (hunk 1 above)
Comment (jiangjiajun) [reply python/tvm/relay/frontend/paddlepaddle.py:1429]: 这里可以把dtype移进去

File: python/tvm/relay/frontend/paddlepaddle.py (hunk 2 above)
Comment (jiangjiajun) [reply python/tvm/relay/frontend/paddlepaddle.py:1441]: 这里由于不是const node，而是从graph获取node，所以只能调用astype

**Pull request closed without merging**